src/core/packages/chrome/navigation/packaging/I18N.md

```diff
@@ -0,0 +1,59 @@
+# Internationalization
+
+## Current Status
+
+**Your labels**: Any language (you provide via props)  
+**Internal strings**: English only
```

Comment:
Going file-by-file 😅 and now I see you did consider those internal strings, so probably there is no easy way to supply them?

Btw there is a task to add more a11y instructions.